app/components/shared/PageWithContainer.js

```diff
@@ -1,15 +1,6 @@
-import React from 'react';
+import classed from './ClassedComponent';
 
-class PageWithContainer extends React.Component {
-  static propTypes = {
-    children: React.PropTypes.node.isRequired
-  };
-
-  render() {
-    return (
-      <div className="container">{this.props.children}</div>
-    );
-  }
-}
+const PageWithContainer = classed('div', 'container');
```

Comment:
Oh I agree! Some of our use of this is a bit ridiculous, so I think it’s worth having the discussion about replacing it, whether with this or with something simpler, because what I keep finding is that each one of these has been implemented with some small use case in mind, and as soon as you try to use it in a more complex one (i.e. add an extra class to one use of a Panel class), you have to add that functionality yourself, and they all get out of sync!

I suspect the more powerful part of this would be allowing it to compose nicely with Styled Components. But as equivalent functionality seems to be on the way, it may not be so necessary! 😜